### PR TITLE
#7344   rename mssql text to ms sql server in browser/data source manager

### PR DESCRIPTION
**Pull request by tlh22**
Changes text as per #7344. Some images may still need to be updated.

### CODE REVIEW

File: docs/user_manual/managing_data_source/opening_data.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ and often write a lot of formats:
   Read the complete list of `supported raster formats
   <https://gdal.org/drivers/raster/index.html>`_.
 * Database formats include PostgreSQL/PostGIS, SQLite/SpatiaLite, Oracle,
-  MSSQL Spatial, SAP HANA, MySQL...
+  MS SQL Server Spatial, SAP HANA, MySQL...
```
Comment (DelazJ) [inline docs/user_manual/managing_data_source/opening_data.rst:33]:
```suggestion
  MS SQL Server, SAP HANA, MySQL...
```

File: docs/user_manual/managing_data_source/opening_data.rst [hunk 2]
```diff
@@ -784,7 +784,7 @@ of them and load their tables:
 
 * |addPostgisLayer| :menuselection:`Add PostGIS Layer...` or by typing
   :kbd:`Ctrl+Shift+D`
-* |addMssqlLayer| :menuselection:`Add MSSQL Spatial Layer`
+* |addMssqlLayer| :menuselection:`Add MS SQL Server Spatial Layer`
```
Comment (DelazJ) [inline docs/user_manual/managing_data_source/opening_data.rst:787]:
```suggestion
* |addMssqlLayer| :menuselection:`Add MS SQL Server Layer`
```

File: docs/user_manual/managing_data_source/opening_data.rst [hunk 3]
```diff
@@ -1060,13 +1060,13 @@ Optionally, you can activate the following checkboxes:
    tables have a **primary key**.
 
 
-.. _create_mssql_connection:
+.. _create_ms_sql_server_connection:
 
-Connecting to MSSQL Spatial
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Connecting to MS SQL Server Spatial
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (DelazJ) [inline docs/user_manual/managing_data_source/opening_data.rst:1066]:
```suggestion
Connecting to MS SQL Server
^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: docs/user_manual/managing_data_source/supported_data.rst [hunk 4]
```diff
@@ -52,7 +52,7 @@ Many of the features and tools available in QGIS work the same,
 regardless the vector data source.
 However, because of the differences in format specifications
 (GeoPackage, ESRI Shapefile, MapInfo and MicroStation file formats,
-AutoCAD DXF, PostGIS, SpatiaLite, Oracle Spatial, MSSQL
+AutoCAD DXF, PostGIS, SpatiaLite, Oracle Spatial, MS SQL Server
 Spatial, SAP HANA Spatial databases and many more), QGIS may handle some of
```
Comment (DelazJ) [inline docs/user_manual/managing_data_source/supported_data.rst:56]:
```suggestion
AutoCAD DXF, PostGIS, SpatiaLite, Oracle Spatial, MS SQL Server,
SAP HANA Spatial databases and many more), QGIS may handle some of
```

File: docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 5]
```diff
@@ -10,7 +10,7 @@
       :local:
 
 QGIS has various capabilities for editing OGR, SpatiaLite, PostGIS,
-MSSQL Spatial and Oracle Spatial vector layers and tables.
+MS SQL Server Spatial and Oracle Spatial vector layers and tables.
```
Comment (DelazJ) [inline docs/user_manual/working_with_vector/editing_geometry_attributes.rst:13]:
```suggestion
MS SQL Server and Oracle vector layers and tables.
```

**Review by DelazJ — COMMENTED** [review]
Thanks @tlh22 for helping.
I made some suggestions below. One thing I'm not sure of is that we used to append "Spatial" to MSSQL as if it was in the name but couldn't find a reference doing a quick search, so I suggest to get rid of it in our docs.

File: docs/user_manual/introduction/qgis_gui.rst [hunk 6]
```diff
@@ -1147,7 +1147,7 @@ copy or paste layer properties (style, scale, CRS...).
      - :guilabel:`Manage Layers`
      - :ref:`label_spatialite`
    * - :menuselection:`-->` |addMssqlLayer|
-       :guilabel:`Add MSSQL Spatial Layer...`
+       :guilabel:`Add MS SQL Server Spatial Layer...`
```
[the remark this answers is not in the document]
Comment (tlh22) [reply docs/user_manual/introduction/qgis_gui.rst:1150]: This is one that could be left with "Spatial" to match Oracle and

File: docs/user_manual/managing_data_source/opening_data.rst (hunk 2 above)
Comment (tlh22) [reply docs/user_manual/managing_data_source/opening_data.rst:787]: This is another one where could match with Oracle and SAP Hana

**Comment by tlh22** [comment]
> Thanks @tlh22 for helping. I made some suggestions below. One thing I'm not sure of is that we used to append "Spatial" to MSSQL as if it was in the name but couldn't find a reference doing a quick search, so I suggest to get rid of it in our docs.

Hi @DelazJ. Happy to try to help - and will try to do more. I thought this was an easy one to start with. Like you, I am not sure about the use of the word Spatial. I was trying to follow the use with Oracle and SAP. Happy for any corrections you see fit. Have suggested places where might be good to match with Oracle/SAP.

**Comment by DelazJ** [comment]
> Happy to try to help - and will try to do more.

Good news.
>  I was trying to follow the use with Oracle and SAP.

The suggestions actually mimicked the tooltip currently used in the "Data Source Manager". These are the button on the "Manage layers" and the entry in "Layer" menu and they open the DSM page so makes sense to align them IMHO.

About "Spatial" suffix, "Spatial" seems to be part of the name of the others GIS module so you can read "Oracle Spatial" or "SAP Hana Spatial" (with capital "s") but what I keep finding is "SQL Server spatial <something>".
https://docs.oracle.com/cd/E28280_01/apirefs.1111/e12048/datacartspatial.htm
https://help.sap.com/docs/r/cbbbfc20871e4559abfd45a78ad58c02/2.0.03/
SQL Server should be treated more like PostgreSQL than the others.
Also the PR I made in code has been merged, meaning that this is what the interface will show from 3.27, and until it is updated, let's align the docs with it.

**Comment by tlh22** [comment]
Thanks @DelazJ. Do you need me to do anything further with the PR or will you handle things from here?

**Review by DelazJ — APPROVED** [review]

**Comment by DelazJ** [comment]
From the "Files Changed" tab, I unresolved the remaining suggestions and commit them.
Should be merged soon.
Thanks again @tlh22

**Pull request merged**